cli/internal/test/builders/network.go

```diff
@@ -0,0 +1,72 @@
+package builders
+
+import (
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/network"
+)
+
+// NetworkResource creates a network resource with default values.
+// Any number of networkResource function builder can be pass to modify the existing value.
+// feel free to add another builder func if you need to override another value
+func NetworkResource(builders ...func(resource *types.NetworkResource)) *types.NetworkResource {
+
+	resource := &types.NetworkResource{
+		Name:       "network resource",
+		ID:         "default id",
+		Driver:     "default driver",
+		Scope:      "default scope",
+		Created:    time.Time{},
+		EnableIPv6: false,
+		Attachable: false,
+		Ingress:    false,
+		ConfigOnly: false,
+		Internal:   false,
+		IPAM:       network.IPAM{},
+		ConfigFrom: network.ConfigReference{},
+		Containers: map[string]types.EndpointResource{"toto": {
```

Comment:
I believe many of these fields are already the default "zero" value, so you don't have to include them here either (false, network.IPAM{},etc)